subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -62,12 +62,7 @@ In order to execute a task called "myTask" on the root project, type:
 $ gradle :myTask
 ----
 
-This will run the single "myTask" and also all of its task <<tutorial_using_tasks.adoc#sec:task_dependencies,dependencies>>.
-
-For convenience, you can also omit the leading ":" from the task name, effectively using a task _selector_ to execute "myTask":
-----
-$ gradle myTask
-----
+This will run the single "myTask" and also all of its <<tutorial_using_tasks.adoc#sec:task_dependencies,task dependencies>>.
```

Comment:
Nit: I wasn't expecting this change as part of this commit, as it just changes the linked text and is not related to the topic of the commit message.

Do you want me to amend my own commit instead? Or should we just leave it as-is now.

Reply:
Sorry about that. I was a bit sloppy there. If you want you can amend this. You can also remove my commit and amend your original one if you prefer that. But you would need do it right now; We want to branch for 7.1 soon. 

But I am also fine to keep this like it is.

Reply:
I've simply squashed our commits and referred to you via `Co-authored-by`.